api_inference_community/validation.py

```diff
@@ -196,8 +198,12 @@ def check_inputs(inputs, tag):
 IMAGE_OUTPUTS = {
     "image-to-image",
     "text-to-image",
+    "latent-to-image",
```

Comment:
Not sure about the new tag, as it maps to the existing `tasks` and I think we probably don't want to create this as an official task while this is experimental. But maybe this is harmless to exist only in the inference API

Reply:
Was added for

https://github.com/huggingface/api-inference-community/blob/329d9bb61a463a9139f318e89619e4ed25fc9080/api_inference_community/routes.py#L229-L241

Seems to be the only usage of `IMAGE_OUTPUTS`

We also need the task in `KNOWN_TASKS` (via `TENSOR_INPUTS`)

https://github.com/huggingface/api-inference-community/blob/329d9bb61a463a9139f318e89619e4ed25fc9080/api_inference_community/routes.py#L96-L102

Should be harmless for this to exist but I agree we don't want it to be an official task while this is experimental.

Reply:
We can maybe find ways to "override" another task for the time being  (without involving another task)